test_project/adam.py

```diff
@@ -101,3 +101,13 @@ def single_iteration():
         result = True
 
     return result
+
+
+def raise_exception():
```

Comment:
The mutator currently change the exception handler and not the raise.
Maybe it is better to change the raise to another one